server/index.js

```diff
@@ -19,6 +19,24 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/', (req, res) => {
+  const sql = `
+    select "productId",
+            "name",
+            "price",
+            "image",
+            "shortDescription"
+    from "products"
+  `;
+  db.query(sql)
+    .then(result => {
+      res.json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
```

Comment:
Here you should pass the error to `next()` like in the `/api/health-check` example